src/bindgen/ir/annotation.rs

```diff
@@ -52,8 +52,15 @@ impl AnnotationSet {
     pub fn load(attrs: &[syn::Attribute]) -> Result<AnnotationSet, String> {
         let lines: Vec<String> = attrs
             .get_comment_lines()
-            .into_iter()
-            .filter(|x| !x.is_empty() && x.starts_with("cbindgen:"))
+            .iter()
+            .filter_map(|line| {
+                let line = line.trim_start();
+                if line.is_empty() || !line.starts_with("cbindgen:") {
```

Comment:
I think we could avoid a bunch of extra allocations here...

Any reason we can't do:

```
let lines = attrs.get_comment_lines();
let lines: Vec<&str> = lines.iter().filter_map(..).collect();
```

Also, the `is_empty()` check looks redundant to me (was redundant already, but I can't see how an empty line can start with `"cbindgen:"` ;))

Reply:
Absolutely no reason! Going to change this asap.

Reply:
Included via rebase.